server/database/models/deployment-logs.ts

```diff
@@ -0,0 +1,43 @@
+import { DatabaseModel } from "@/server/database/lib/general/database-model";
+import { z } from "zod";
+
+export class DeploymentLog {
+  constructor(
+    readonly id: string,
+    readonly commitHash: string,
+    readonly createdAt: Date,
+  ) {}
+}
+
+export class DeploymentLogModel extends DatabaseModel<
+  DeploymentLog,
+  string,
+  z.input<typeof DeploymentLogModel.schema>
+> {
+  static instance = new DeploymentLogModel();
+
+  private static schema = z.object({
+    commitHash: z.string(),
+    createdAt: z.coerce.date(),
```

Comment:
```suggestion
    createdAt: z.date(),
```
I think `coerce` is intended for converting from strings, however the DB should be storing these as dates for us, so probably not needed here.